django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -121,10 +121,16 @@ async def receive_llm_responses(
     async def handle_documents(
```

Comment:
Is it worth adding a docstring here? It's not the easiest method to make sense of (particularly `files_and_citations` and an explanation of why we're sending something different to the client than we're returning)

Reply:
yep, good point

Reply:
[b06110f](https://github.com/i-dot-ai/redbox/pull/947/commits/b06110fc73929c7f013e35d569266f9b51ec568e)